### PR TITLE
gstreamer, gst-* 1.22.0

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
let's use PR to merge all the formulae. see [this comment](https://github.com/Homebrew/homebrew-core/pull/113061#issuecomment-1279296473)

**Comment by p-linnane** [comment]
Audit failure for `gst-python`:
```
gst-python:
    * python modules have explicit framework links
      These python extension modules were linked directly to a Python
      framework binary. They should be linked with -undefined dynamic_lookup
      instead of -lpython or -framework Python.
        /opt/homebrew/opt/gst-python/lib/python3.11/site-packages/gi/overrides/_gi_gst.cpython-311-darwin.so
```
Aside from that, I'm not sure how to consolidate all these formulae. CC @Homebrew/core for additional thoughts.

**Comment by SMillerDev** [comment]
If someone wants to do that: good. But doesn't have to stop a version bump.

**Review by SMillerDev — DISMISSED** [review]

File: Formula/gst-python.rb [hunk 1]
```diff
@@ -27,12 +27,6 @@ class GstPython < Formula
   depends_on "pygobject3"
   depends_on "python@3.11"
 
-  # See https://gitlab.freedesktop.org/gstreamer/gst-python/-/merge_requests/41
-  patch do
-    url "https://gitlab.freedesktop.org/gstreamer/gst-python/-/commit/3e752ede7ed6261681ef3831bc3dbb594f189e76.diff"
-    sha256 "d6522bb29f1894d3d426ee6c262a18669b0759bd084a6d2a2ea1ba0612a80068"
-  end
```
Comment (cho-m) [inline Formula/gst-python.rb:34]: This patch needs to be updated not removed. Someone may want to take over upstream MR and submit to monorepo.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**